README.md

```diff
@@ -1,9 +1,12 @@
 # Running Gitpod in [Amazon EKS](https://aws.amazon.com/en/eks/)
 
-> **IMPORTANT** This guide exists as a simple and reliable way of creating a Gitpod instance in Amazon EKS. It
+> **IMPORTANT** This guide exists as a simple and reliable way of creating required AWS infrastructure. It
```

Comment:
I'd change this to "This guide exists as a simple and reliable way of creating an environment in AWS (EKS) that Gitpod can be installed into."